packages/components/src/ProgressBar/ProgressBar.tsx

```diff
@@ -26,20 +26,12 @@ export function ProgressBar({
   totalSteps,
   size = "base",
 }: ProgressBarProps) {
-  const percentage = ((currentStep - 1) / (totalSteps - 1)) * 100;
-  const widthPercent = Math.min(100, Math.max(0, percentage));
-  const progressBarContainerClassName = classnames(
-    styles.container,
-    sizes[size],
-  );
+  const percentage = (currentStep / totalSteps) * 100;
```

Comment:
I think I could actually remove this, and change the accessibility text to `{currentStep} of {totalSteps} complete`

Reply:
Conversation: will not remove, because this actually allows the accessibility text to more accurately reflect the information the ProgressBar conveys